src/test/scala/com/github/tototoshi/slick/JodaSupportSpec.scala

```diff
@@ -224,6 +225,41 @@ abstract class JodaSupportSpec extends AnyFunSpec
     res2.lift(1).map(_.localDate) should not be Some(new LocalDate(2012, 12, 5))
     res2.lift(2).map(_.localDate) should not be Some(new LocalDate(2012, 12, 5))
   }
+
+  private val systemTimeZone = java.util.TimeZone.getDefault().getID
+  Seq("Asia/Tokyo", "UTC", "Europe/London", "America/New_York").foreach { tz =>
+    val timeZone = DateTimeZone.forID(tz)
+    it(s"should be the same in/out joda-time zoned $tz") {
+      val otherZonedDateTime = new DateTime(2012, 12, 7, 0, 0, 0, 0, timeZone)
+      val data = Jodas(
+        DateTimeZone.forID("Asia/Tokyo"),
+        new LocalDate(2012, 12, 7),
+        otherZonedDateTime,
+        new DateTime(2012, 12, 7, 0, 0, 0, 0, DateTimeZone.UTC).toInstant,
+        new LocalDateTime(2012, 12, 7, 0, 0, 0, 0),
+        new LocalTime(1000),
+        Some(DateTimeZone.forID("America/New_York")),
+        Some(new LocalDate(2012, 12, 6)),
+        None,
+        Some(new Instant(1000)),
+        None,
+        Some(new LocalTime(1000))
+      )
+      db.run(jodaTest += data).await()
+
+      val actual = db.run(jodaTest.result).await()
+      actual should have size 1
+
+      actual.foreach { d =>
+        if (isWithoutCalender)
+          d.dateTime.getMillis should be(data.dateTime.getMillis)
```

Comment:
If `isWithoutCalender == true` then every test will be successful in all timezone.